sandboxed_containers/sandboxed-containers-4.9-release-notes.adoc

```diff
@@ -0,0 +1,83 @@
+
+* Previously, creating multiple `KataConfig` custom resources resulted in a silent failure, with no error from the {product-title} web console notifying the user that creating more than one custom resource failed. With this release, the user receives an error when trying to create multiple custom resources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2019381[*BZ#2019381*])
+
+* Previously, there were instances where the Operator Hub in the {product-title} web console did not display icons for an Operator. With this release, icons are always displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2019380[*BZ#9019380*])
+
+[id="sandboxed-containers4-9-known-issues"]
+== Known issues
+
+* If you are using {sandboxed-containers-first}, you cannot use the `hostPath` volume in a {product-title} cluster to mount a file or directory from the host node’s file system into your pod. As an alternative, you can use local persistent volumes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+[id="sandboxed-containers-4-9-asynchronous-errata-updates"]
+== Asynchronous errata updates
+
+Security, bug fix, and enhancement updates for {sandboxed-containers-first} {product-version} are released as asynchronous errata through the Red Hat Network. All {product-title} {product-version} errata is https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle] for more information about asynchronous errata.
+
+Red Hat Customer Portal users can enable errata notifications in the account settings for Red Hat Subscription Management (RHSM). When errata notifications are enabled, users are notified via email whenever new errata relevant to their registered systems are released.
+
+[NOTE]
+====
+Red Hat Customer Portal user accounts must have systems registered and consuming {product-title} entitlements for {product-title} errata notification emails to generate.
+====
+
+This section will continue to be updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {sandboxed-containers-first} {product-version}.
+
+[id="sandboxed-containers-1-1-0"]
+=== RHEA-2021:80949 - {sandboxed-containers-first} 1.1.0 image release, bug fix and enhancement advisory
+
+Issued: 2021-10-21
+
+{sandboxed-containers-first} release 1.1.0 is now available. This advisory contains an update for {sandboxed-containers-first} with enhancements and bug fixes.
+
+The list of bug fixes included in the update is documented in the link:https://access.redhat.com/errata/product/290/ver=4.9/rhel---8/x86_64/RHEA-2021:3941[RHEA-2021:80949] advisory.
```

Comment:
I would suggest using this link instead: https://access.redhat.com/errata/RHEA-2021:3941